pkg/relayer/proxy/synchronous.go

```diff
@@ -90,6 +91,7 @@ func (sync *synchronousRPCServer) Stop(ctx context.Context) error {
 // when synchronousRPCServer is used as an http.Handler with an http.Server.
 // (see https://pkg.go.dev/net/http#Handler)
 func (sync *synchronousRPCServer) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
```

Comment:
I can't see where the metrics endpoint is started on the `RelayMiner`, Am I missing something?

Reply:
Same set up as AppGate: https://github.com/pokt-network/poktroll/blob/issue/24-relayminer-appgateserver-prometheus/pkg/relayer/relayminer.go#L81